utils/config.py

```diff
@@ -31,9 +31,9 @@ def load_config_from_env() -> None:
     logger_config = config_model.LoggerConfig()
     ols_config.logger_config = logger_config
 
-    logger_config.default_level = os.getenv("LOG_LEVEL", logging.INFO)
+    logger_config.default_level = int(os.getenv("LOG_LEVEL", logging.INFO))
```

Comment:
There is `LOG_LEVEL=INFO` in `default.env`. That wouldn't work with this. You can set the logging level as string or int, it can work with both, so make sure we choose just one way of setting it. For me, it's a bit more visible to have debug/info/,... instead of numeric values.

Reply:
ok in this case we need to update it somewhere else, yes. Lemme do it

Reply:
fixed @onmete